django/http/multipartparser.py

```diff
@@ -551,8 +584,9 @@ def _find_boundary(self, data, eof=False):
         """
         Finds a multipart boundary in data.
 
-        Should no boundary exist in the data None is returned instead. Otherwise
```

Comment:
I don't think touching docstrings for wraps at 80 characters instead of 79 is worth the churn.